grobid_client/grobid_client.py

```diff
@@ -253,7 +253,9 @@ def process_pdf(
         include_raw_citations,
         include_raw_affiliations,
         tei_coordinates,
-        segment_sentences
+        segment_sentences,
+        start=-1,
+        end=-1
```

Comment:
@lfoppiano you'll need to pass the two new arguments in the next call to the method for response code 503

Reply:
Indeed. Sorry!!